.golangci.yml

```diff
@@ -85,16 +86,24 @@ linters:
     - nosnakecase
     - paralleltest
     - prealloc
+    - rowserrcheck
     - scopelint
+    - sqlclosecheck
+    - structcheck
     - tagliatelle
     - testpackage
     - tparallel
+    - varcheck
     - varnamelen # this one may be interesting, but too much churn
+    - wastedassign
     - whitespace
 
 issues:
   exclude-use-default: false
   exclude-rules:
+    - path: \.pb\.go
```

Comment:
Nah I would basically encourage go fmt before reach commit here. Typically having a `make presubmit` command that line and formats is a good way to avoid needing these rules

Reply:
That seems like a good idea. Can you push a PR with the approach you are suggesting?
OTOH, it seems OK to have exceptions for generated code.